portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/Translation.path

```diff
@@ -44,6 +44,11 @@
 	<td>//div[contains(@class,'localized')]//button[contains(@class,'localized') and contains(@id,'description')][*[normalize-space(text())='${key_locale}']]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>DROPDOWN_MENU_ITEM</td>
```

Comment:
Is this necessary? Can we use ``MenuItem#ANY_MENU_ITEM``?

Reply:
Or ``MenuItem#DROPDOWN_MENU_ITEM``? We should try to reuse when we can

Reply:
I have tried these two paths, they are not working for this test.